tests/migration_test.go

```diff
@@ -2719,26 +2708,12 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 					node, err = virtClient.CoreV1().Nodes().Get(context.Background(), vmi.Status.NodeName, metav1.GetOptions{})
 					Expect(err).ToNot(HaveOccurred())
 
-					originalNodeLabels = copyMap(node.Labels)
-					originalNodeAnnotations = copyMap(node.Annotations)
-
-					node = disableNodeLabeller(node, virtClient)
+					node = stopNodeLabeller(node.Name, virtClient)
 				})
 
 				AfterEach(func() {
-					By("Restore node to its original state")
-					node.Labels = originalNodeLabels
-					node.Annotations = originalNodeAnnotations
-					node, err = virtClient.CoreV1().Nodes().Update(context.Background(), node, metav1.UpdateOptions{})
-					Expect(err).ShouldNot(HaveOccurred())
-
-					Eventually(func() map[string]string {
-						node, err = virtClient.CoreV1().Nodes().Get(context.Background(), node.Name, metav1.GetOptions{})
-						Expect(err).ShouldNot(HaveOccurred())
-
-						return node.Labels
-					}, 10*time.Second, 1*time.Second).Should(Equal(originalNodeLabels), "Node should have fake host model")
-					Expect(node.Annotations).To(Equal(originalNodeAnnotations))
+					By("Resuming node labeller")
+					node = resumeNodeLabeller(node.Name, virtClient)
```

Comment:
please verify that `node.Labels[v1.HostModelCPULabel+"fake-model"]` doesn't exist.

Actually i think it exist after the test because in:

https://github.com/kubevirt/kubevirt/blob/970d68ae81247e082c708d54fbf14cfb70c728bb/pkg/virt-handler/node-labeller/node_labeller.go#L305

the labeller never delete it ,this might cause flackiness in later tests and it will be VERY hard to catch this.

Reply:
Done. Thanks!!